notebooks/growth_model_forecast.ipynb

```diff
@@ -7,14 +7,23 @@
    "source": [
```

Comment:
<p>Since we are now using a weekly dataset, we should update this to mention <code>number of weeks an account has non-zero revenue....</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>